src/components/ebay-notice/README.md

```diff
@@ -0,0 +1,22 @@
+# ebay-notice
+
+## `ebay-notice` tag
+The `<ebay-notice>` is a tag used to create a custom-designed notice element. The notice can be single or multi-line but each line should be wrapped inside a `<p>` tag.
+
+### `ebay-notice` Usage
+
+```marko
+<ebay-notice heading-tag="h3" type="priority" level="page" aria-label-text="this is a notice">
```

Comment:
(minor) Would "this is a notice" be used in real world usage of this component? If not, maybe we can improve it to approximate practical usage (across all examples that use this text).

Reply:
Right, I will change it to @ianmcburnie's suggestion